Stack/Opc.Ua.Core/Stack/State/NodeState.cs

```diff
@@ -885,7 +885,7 @@ public virtual AttributesToSave GetAttributesToSave(ISystemContext context)
                 attributesToSave |= AttributesToSave.WriteMask;
             }
 
-            if (m_writeMask != AttributeWriteMask.None)
+            if (m_userWriteMask != AttributeWriteMask.None)
```

Comment:
Not clear why you think this is a error.

Reply:
Commit updated. The function is checking the WriteMask and UserWriteMask but here there was a mix of WriteMask + UserAccessLevel instead of UserWriteMask only

Reply:
Hi @opcfoundation-org, my understanding from your comment is now the following:
i) A writemask != None triggers to write the UserAccessLevel
ii) UserWriteMask is never written per se, none of the code sets the AttributesToSave flag, but
iii) intentionally the userwritemask is written if writemask != None with the value of the writemask on Save 
Please advise

Reply:
We were able to resolve the misunderstandings and the PR looks good as is.